### ISSUE TITLE
Very low FPS rate (Windows 11)

### ISSUE THREAD

**Issue by brunosaboia** [OldUnreal/UnrealTournamentPatches#1385]
Hello,

I tried running UT with this patch on my MacBook M1 and works like a charm.

However, I tried to run on my laptop that runs Windows 11 now, and surprisingly it is really sluggish. No matter what setting I pick, the game is low on FPS. For some reason, the best video mode I could find seemed to be Software Rendering, but honestly it is not very good also.

This hardware is enough to run more modern games, like StarCraft II, with all settings on high and no lag at all.

I installed the latest patch available to date, OldUnreal-UTPatch469d-Windows.exe. My install is a GOTY CD.

My hardware specs:

```
Processor            Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz   1.80 GHz
Installed RAM        8.00 GB (7.89 GB usable)
System type          64-bit operating system, x64-based processor
Video                NVIDIA GeForce MX150
```

If that matters, it is a laptop. [This exact model](https://www.techradar.com/reviews/xiaomi-mi-notebook-pro).

Although this PC is far from being a modern high-end computer, it should run a game from 1999 without any sweat.

Can anyone help?

Thanks!

**Comment by SeriousBuggie** [comment]
Did you try all major renders? OpenGL, XOpenGL, D3D9?

**Comment by brunosaboia** [comment]
Hello @SeriousBuggie, yes, I tried all of them. XOpenGL doesn't work at all, the others are similarly slow.

**Comment by an-eternity** [comment]
I have also noticed some rendering performance issues with v469d-rc2, but had no time to look into it so far.
In my case a framerate just drops from 100+ down to 30 or lower for a little while with no any obvious reason.
(Testing with D3D9, AMD graphics, Windows 11)

**Comment by garry-ut99** [comment]
Other players also noticed recently some rendering performance issues, they might be related to each other, or not, but it's worth to mention about them in case they turn out to be related and helpful for others to solve their own similiar issues :
`https://github.com/OldUnreal/UnrealTournamentPatches/issues/1381`
`https://github.com/OldUnreal/UnrealTournamentPatches/issues/1356`

**Comment by garry-ut99** [comment]
> Does this laptop also have an integrated GPU?

This part of the question is unnecessary, because he already provided such info, his CPU model : [LINK](https://www.intel.com/content/www/us/en/products/sku/124967/intel-core-i58250u-processor-6m-cache-up-to-3-40-ghz/specifications.html#specs-1-0-4) and laptop model : " [This exact model](https://www.techradar.com/reviews/xiaomi-mi-notebook-pro) " which both links says it has `Intel® UHD Graphics 620` integrated.

**Comment by guitronimo** [comment]
Morning, 
as stijn-volckaert suggests, check if it runs on your Dedicated GPU. 
I had same issue. Unfortunately not a fix yet found for XOpenGL & D3D. But for OpenGL. 

If your dedicated GPU runs ~ 0% then its likey that the game runs on your UHD only.

You can try this:
[Engine.Engine]
GameRenderDevice=OpenGLDrv.OpenGLRenderDevice

[OpenGLDrv.OpenGLRenderDevice]
PreferDedicatedGPU=True

**Comment by brunosaboia** [comment]
Hello everyone, thanks for the valuable hints. I now fixed it and I will say how for posterity, perhaps it will help someone else.

Following the tip by @stijn-volckaert, I saw that indeed the GPU was not being used. I tried the config file change suggested by @guitronimo, but that by itself did not help, it was still not using the GPU.

I then realized that only the NVIDIA GPU drivers were installed. I installed the integrated Intel one as well, and now it works like a charm.

Thanks for all the help.
